tasks/install-ubuntu.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: add Foreman deb repo
+  apt_repository:
+    repo: deb http://deb.theforeman.org/ trusty stable
```

Comment:
Please consider adding this code to [kostyrevaa.foreman](https://github.com/kostyrevaa/ansible-role-foreman)